### DISCUSSION TITLE
Church YOURLS install needs help.. white screen...

### DISCUSSION THREAD

**Discussion by journeyguy** [YOURLS/YOURLS#3566] [Q&A]
Hey folks...

We use YOURLS everywhere for our church. I logged in today and got the white screen of death. We recently got a message from Dreamhost informing us our sites had been upgraded to PHP v8.1.

Thinking that we needed to upgrade YOURLS (we were running an older version), I upgraded to 1.9.2. Saw the login window confirming it had been upgraded, but when I logged in... white screen again.

We had a former member install for us, and it's a super busy season right now. Would any of you be willing to take a look (we will compensate you) and get things up and running again? Please and thank you!

**Comment by ehthxC7x9D** [comment]
@journeyguy I am familiar enough with shared hosting and YOURLS that I think I could help debug via email or a phone call.  jarebear6expepjozn6rakjq5iczi3irqwphcvbswgkahd6b6twnxxid@4xvk.com feel free to reach out.

**Reply by journeyguy** [reply]
Thank you! Emailed you.
